packages/rfq-loadtest/config-template.yaml

```
@@ -0,0 +1,69 @@
+
+# approx how much ETH to send on each test bridge? 0.00007
+TEST_BRIDGE_AMOUNT_UNITS: 0.00007
+
+# trigger a rebalance when gas reaches this value or lower
+MINIMUM_GAS_UNITS: 0.003
+
+# when a rebalance is triggered, resupply the deficit chain to this amount
+REBALANCE_TO_UNITS: 0.01
+
+CHAINS:
+  10: 
```

yamllint

[error] trailing spaces

(trailing-spaces)